seqr/views/apis/staff_api_2_3_tests.py

```diff
@@ -740,7 +745,9 @@ def test_upload_qc_pipeline_output(self, mock_file_iter):
         mock_file_iter.return_value = SAMPLE_QC_DATA_MORE_DATA_TYPE
         response = self.client.post(url, content_type='application/json', data=request_data)
         self.assertEqual(response.status_code, 400)
-        self.assertEqual(response.reason_phrase, 'Multiple dataset types detected: wes ,wgs')
```

Comment:
Just change the error message generation to sort these

Reply:
Changed.